### DISCUSSION TITLE
Installation command issue - Apptainer

### DISCUSSION THREAD

**Discussion by eMKiso** [nextpyp/.github#74] [General]
Dear developers,

in the installation instructions, [Step 2](https://nextpyp.app/files/pyp/latest/docs/install/install-web.html#step-2-install-operating-system-packages), the command 
`sudo apt-get install -y ./apptainer_1.1.0-rc.2_amd64.deb` did not work for me on Ubuntu 20.04.
However if one changes `sudo apt-get install ...` to `sudo apt install ...` then it works. Is this a mistake in the instructions or is it different in Ubuntu 22.04?

For beginners, it might also be useful to add the chmod command to mark the apptainer .deb file as executable, just like it is for the installation script in the [Step 3](https://nextpyp.app/files/pyp/latest/docs/install/install-web.html#step-3-download-and-run-the-installation-script).

Best regards!

**Comment by cuchaz** [comment]
Are you using Ubuntu 20.04? `apt-get` is usually installed by default on Ubuntu images, especially the older ones.

You can use `apt` just fine, but it would be weird if `apt-get` weren't available too. That usually indicates some kind of OS installation error or misconfiguration issue.

For what it's worth, we should probably change the instructions to use `apt` anyway, since that's supposed the newer human-friendly CLI interface to the package manager.

Also, as far as I know, `.deb` files do not need to be marked executable to install them. Is that required on your system too?

**Comment by eMKiso** [comment]
Hi,

my bad. I made multiple mistakes.

1. I almost always use key 'Tab' on the keyboard to autocomplete the names in the CLI. The autocomplete function in my shell doesn't work with `apt-get` but works with `apt`. My mistake. The `apt-get` is present on our system as per usual.
2. Before figuring out the point above, I assumed (wrongly) that the .deb should be executable to be able to use autocomplete with `apt-get`. Basically it doesn't matter, if .deb is executable or not, you are correct.

Sorry for the false alarm and thank you for the quick reply!

Best!
